server/planning/events.py

```diff
@@ -435,3 +441,14 @@ def generate_recurring_dates(start, frequency, interval=1, until=None, byday=Non
         return (tz.localize(dt).astimezone(pytz.UTC).replace(tzinfo=None) for dt in dates)
     else:
         return (date for date in dates)
+
+
+def setRecurringMode(event):
```

Comment:
This cannot be done on the client side ? We try to handle data as much as we can on the client side